english/normal-dungeon/4/1.md

```diff
@@ -0,0 +1,5 @@
+As you go into the room of the 🏠house, your wife👧🏻 is crying and tells you how she managed to escape and hide from the 👺goblins. She also told you that there is a secret mountain⛰️ where the goblins is keeping your son👦
```

Comment:
What if, a female player is playing this game ?

Reply:
Good question, since it is a dungeon game, you can be any character : )

Reply:
Also, The wife's character was already in the game.